layers/+source-control/version-control/packages.el

```diff
@@ -105,14 +106,15 @@
 
 (defun version-control/init-git-gutter+ ()
   (use-package git-gutter+
-    :commands (global-git-gutter+-mode git-gutter+-mode)
+    :commands (global-git-gutter+-mode git-gutter+-mode git-gutter+-refresh)
```

Comment:
Are there any benefits to adding an autoload for `git-gutter+-refresh`? I don't see any...